src/main/java/com/b44t/messenger/rpc/Rpc.java

```diff
@@ -32,7 +32,7 @@ private void processResponse() throws JsonSyntaxException {
         String jsonResponse = dcJsonrpcInstance.getNextResponse();
         Response response = gson.fromJson(jsonResponse, Response.class);
 
-        if (response.id == 0) { // Got JSON-RPC notification/event, ignore
+        if (response == null || response.id == 0) { // Got JSON-RPC notification/event, ignore
```

Comment:
If we merge this, then we definitely need to log an error if `response == null`. Otherwise, DC may just hang on startup forever without any helpful message in the logs.

Reply:
added a commit